src/main/webapp/contact.html

```diff
@@ -1,29 +1,104 @@
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+    <!-- js -->
+    <script src="./assets/js/mainScript.js"></script>
   </head>
   <body>
     <!-- nav bar -->
     <header>
       <div class="logo"><a href="/">noms</a></div>
       <ul class="nav">
-          <li class="navlink"><a href="/about.html">about</a></li>
-          <li class="navlink"><a href="/contact.html">contact</a></li>
-          <li class="navlink"><a href="/sign-up.html">sign up</a></li>
+        <li class="navlink"><a href="/about.html">about</a></li>
+        <li class="navlink"><a href="/contact.html">contact</a></li>
+        <li class="navlink"><a href="/sign-up.html">sign up</a></li>
       </ul>
       <div class="burger">
-          <i class="fas fa-bars"></i>
+        <i class="fas fa-bars"></i>
       </div>
     </header>
-    <h1 class="title">contact page to be added</h1>
+    <!-- main content -->
+    <div id="col-container">
+      <h1 id="title">contact</h1>
+        <div id="page-container">
+          <div id="map-container">
+            <div id="map">
+            </div>
+          </div>
+          <div id="podmates">
+            <div class="mate">
+              <div class="basic-info">
+                <div class="row">
+                  <img src="./assets/jpeg/uma.jpeg" alt="Picture of Uma">
+                  <div class="col">
+                    <h4>Uma Bahl | Santa Clara University | c/o 2022</h4>
+                    <div class="row" style="text-align: center;">
+                      <a href="http://github.com/ubahl/">
+                        <img src="./assets/svg/github.svg" alt="Image of the GitHub logo"> ubahl
+                      </a>
+                      <a href="https://www.linkedin.com/in/umabahl/">
+                        <img src="./assets/svg/linkedin.svg" class="svg" alt="Image of the LinkedIn logo"> umabahl
+                      </a>
+                    </div>
+                    <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                      labore et dolore magna aliqua.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="mate">
+              <div class="basic-info">
+                <div class="row">
+                  <img src="./assets/jpeg/melanie.jpeg" alt="Picture of Melanie">
+                  <div class="col">
+                    <h4>Melanie Gutzmann | Oregon State University | c/o 2022</h4>
+                    <div class="row">
+                      <a href="http://github.com/mirrorkeydev/">
+                        <img src="./assets/svg/github.svg" alt="Image of the GitHub logo"> mirrorkeydev
+                      </a>
+                      <a href="https://www.linkedin.com/in/mgutzmann/">
```

Comment:
is actually https://www.linkedin.com/in/mirrorkeydev/ nowadays :)

Reply:
Nice! Will change :)